std/assembly/memory.ts

```diff
@@ -46,8 +46,44 @@ export namespace memory {
   // @ts-ignore: decorator
   @unsafe
   export function repeat(dst: usize, src: usize, srcLength: usize, count: usize): void {
-    var index: usize = 0;
-    var total = srcLength * count;
+    let index: usize = 0;
+    let total = srcLength * count;
+
+    if (ASC_SHRINK_LEVEL < 1) {
+      switch (<u32>srcLength) {
+        case 0: return;
+        case 1: {
+          memory.fill(dst, load<u8>(src), count);
+          return;
+        }
+        case 2: {
+          let srcVal = load<u16>(src);
+          while (index < total) {
+            store<u16>(dst + index, srcVal);
```

Comment:
The stores here are potentially unaligned, which makes me wonder whether `memory.copy`, especially when using the intrinsic, could perform better?

Reply:
Good catch. I will force it to `1 byte` alighment via hint. But I thinking about rewrite all of this to repeat by squaring approach. And probably this cases will not need anymore